var/spack/repos/builtin/packages/subversion/package.py

```diff
@@ -60,12 +66,30 @@ def install(self, spec, prefix):
         options.append('--with-zlib=%s' % spec['zlib'].prefix)
         options.append('--with-sqlite=%s' % spec['sqlite'].prefix)
         options.append('--with-serf=%s' % spec['serf'].prefix)
-        # options.append('--with-swig=%s' % spec['swig'].prefix)
+
+        if spec.satisfies('^swig'):
+            options.append('--with-swig=%s' % spec['swig'].prefix)
+        if spec.satisfies('+perl'):
+            options.append(
+                'PERL=%s' % join_path(spec['perl'].prefix.bin, 'perl'))
 
         configure(*options)
         make()
+        if self.run_tests:
+            make('check')
         make('install')
 
+        if spec.satisfies('+perl'):
+            make('swig-pl')
+            if self.run_tests:
+                make('check-swig-pl')
+            make('install-swig-pl-lib')
+            with working_dir(join_path(
+                    'subversion', 'bindings', 'swig', 'perl', 'native')):
+                perl = Executable(join_path(spec['perl'].prefix.bin, 'perl'))
```

Comment:
This can just be `perl = which('perl')`, as the perl dependency's `bin` directory will be added to `PATH` during build.